dask/dataframe/multi.py

```diff
@@ -927,7 +928,11 @@ def concat_indexed_dataframes(dfs, axis=0, join="outer"):
 
 def stack_partitions(dfs, divisions, join="outer"):
     """Concatenate partitions on axis=0 by doing a simple stack"""
-    meta = methods.concat([df._meta for df in dfs], join=join, filter_warning=False)
+    meta = make_meta(
+        methods.concat(
```

Comment:
Can you include a comment pointing to https://github.com/pandas-dev/pandas/issues/32934?

Reply:
> I don't recall seeing the CI failure before: https://travis-ci.org/github/dask/dask/jobs/670481541?utm_medium=notification&utm_source=github_status. Does it look related?

Tests ran through locally. No idea whats going on here to be honest.